bin/yaml/libraries.yaml

```diff
@@ -370,7 +370,8 @@ libraries:
       subdir: vcl
       target_prefix: v
       targets:
-        - 2.00.01
+        - 2.00
```

Comment:
whoops! I see you edited this to 2.00 (from 2.00.01); I'll revert this else things go funny downstream

Reply:
(sorry for not catching in the review; my fault)